classes/submission/reviewAssignment/ReviewAssignmentDAO.inc.php

```diff
@@ -376,15 +375,21 @@ function insertObject($reviewAssignment) {
 		);
 
 		$reviewAssignment->setId($this->getInsertId());
-		return $reviewAssignment->getId();
+
+		// Update review stage status whenever a review assignment is changed
+		if ($result) {
+			$this->updateReviewRoundStatus($reviewAssignment);
+		}
+
+		return $reviewAssignment;
 	}
 
 	/**
 	 * Update an existing review assignment.
 	 * @param $reviewAssignment object
 	 */
```

Comment:
Looks lie it was unused anyway. Removed the return value: https://github.com/NateWr/pkp-lib/commit/f6863d668a1dcd8d7cd7eb2e09d5ed38b3a76c9a